TOPdeskPS/functions/Send-TdWebDAVFile.ps1

```diff
@@ -56,28 +55,27 @@
         $Credential,
 
         [Parameter(Mandatory = $true)]
-        [ValidateSet('accesslogs', 'csvexport', 'database_backup', 'import', 'photos', 'topsis', 'upload', 'web')]
-        [System.IO.FileInfo]
+        [ValidateSet('accesslogs', 'csvexport', 'database_backup', 'import', 'photos', 'photos/items', 'photos/locations', 'photos/objects', 'photos/operators', 'photos/others', 'photos/persons', 'photos/sites', 'topsis', 'upload', 'web')]
+        [string]
         $Folder,
 
         [Alias('TOPdeskUrl')]
        	[PSFValidatePattern('http(s)?://([\w-]+\.)+[\w-]+(/[\w- ./?%&=]*)?', ErrorMessage = '{0} is not a valid TOPdesk Url.')]
         [System.String]
-        $Url = (
-            Get-PSFConfigValue -FullName TOPdeskPS.Url -NotNull -ErrorAction Continue)
+        $Url = (Get-PSFConfigValue -FullName TOPdeskPS.Url -NotNull -ErrorAction Continue)
     )
 
     begin {
+$Folder
```

Comment:
I don't think that we need to output this string.

Reply:
Dunno how that got left in there! Removed! :)